cmd/yggctl/main.go

```diff
@@ -97,6 +98,58 @@ from stdin.`,
 					},
 					Action: generateControlMessageAction,
 				},
+				{
+					Name:      "worker-data",
+					Usage:     "Generate data files needed for workers to interact with yggd",
+					UsageText: "yggctl generate worker-data [command options]",
+					Description: `The generate worker-data command creates data files necessary for workers to
+communicate properly with yggd.`,
+					Flags: []cli.Flag{
+						&cli.BoolFlag{
+							Name:    "install",
+							Aliases: []string{"i"},
+							Usage:   "install data files into system-appropriate directories",
+						},
+						&cli.PathFlag{
+							Name:    "output",
+							Aliases: []string{"o"},
+							Usage:   "output files to `DIRECTORY`",
+						},
+						&cli.StringFlag{
+							Name:     "name",
+							Aliases:  []string{"n"},
+							Usage:    "generate files using `NAME`",
+							Required: true,
+						},
+						&cli.StringFlag{
+							Name:     "program",
+							Aliases:  []string{"p"},
+							Usage:    "set the worker program to `PATH`",
+							Required: true,
+						},
+						&cli.StringFlag{
+							Name:     "user",
+							Aliases:  []string{"u"},
+							Usage:    "set the worker user to `USER`",
+							Required: true,
+						},
```

Comment:
What about adding arbitrary `StringFlag` "group"? because user could be different in some case from the group.

Reply:
Ah yea, good idea.